### PR TITLE
Live 1408: Interview Byline

### PR DESCRIPTION
**Pull request by abeddow91**
## Why are you doing this?
This pr adds a bottom border to the Editions byline component.

## Changes

- Adds borderWidthStyles 
- Adds bottom border to byline for Interview
- Replaces px with remSpace in byline

## Screenshots

| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

**Comment by joecowton1** [comment]
Are we still missing a border-right on the byline? 😬

**Comment by abeddow91** [comment]
Thanks for raising @joecowton1. Interview designs are inconsistent for right byline borders but I've added here as this seems to be the case across the rest of the templates and I suspect omitting the right border on the  wide device template is an mistake.

File: src/components/editions/byline.tsx [hunk 1]
```diff
@@ -18,26 +19,28 @@ import type { FC, ReactNode } from 'react';
...
+		box-sizing: border-box;
+		margin-left: ${remSpace[6]};
+		padding-left: ${remSpace[3]};
+		padding-right: ${remSpace[3]};
 	}
 
 	${from.wide} {
-		margin-left: ${wideArticleMargin}px;
+		margin-left: ${remSpace[24]};
+		left: ${remSpace[12]};
```
Comment (joecowton1) [inline src/components/editions/byline.tsx:39]: This looks a bit suspect -  can we not just use the margin-left at `9rem` or even the value from the `styles` files that we've used elsewhere?

Comment (abeddow91) [reply src/components/editions/byline.tsx:39]: Think this was just to favour remSpace but agreed, looks gnarly. Will revert this!

**Review by joecowton1 — APPROVED** [review]

**Pull request merged**